http/README.md

```diff
@@ -278,3 +278,19 @@ deleteCookie(headers, "deno", { path: "/", domain: "deno.land" });
 ```
 
 > Note: At the moment multiple `Set-Cookie` in a `Response` is not handled.
+
+### getSetCookies
+
+### getCookies
```

Comment:
This heading looks like a mistake?

Reply:
No, I named it `getSetCookies` deliberately. I understand the possible confusion given both `get` and `set` prefixes in the same function name, but I believe it makes sense once its purpose is made clear.

[Go's cookie.go](https://github.com/golang/go/blob/master/src/net/http/cookie.go) uses the alike `readSetCookies`, which makes sense to me, personally.

Perhaps, `parseSetCookies` might be a better-suited name. What do you think?

Reply:
`getSetCookies` looks fine to me, but there's another subsection with the title of `getCookies` here. Is this intentional?

Reply:
Wow, I completely misread that! Apologies. Fixed now 😅